examples/11_advanced_parametric_study.py

```diff
@@ -25,18 +25,16 @@
 
 This example shows how you can use PyAdditive to perform a parametric study.
 The intended audience is a user who desires to optimize additive machine parameters
-to achieve a specific result. Here, the ``ParametricStudy`` class is used to
-conduct a parametric study. This is not required but ``ParametricStudy``
+to achieve a specific result. Here, the :class:`ParametricStudy` class is used to
+conduct a parametric study. This is not required but :class:`ParametricStudy`
 provides data management and visualization features which ease the task.
```

Comment:
```suggestion
provides data management and visualization features that ease the task.
```
Restrictive clauses (which are essential) use that.
Non-restrictive clauses (which are optional) use which and are preceded by a comma.